postfix/templates/default/generic.erb

```diff
@@ -0,0 +1,3 @@
+<% @my_destination.each do |host| -%>
```

Comment:
Basically a catch-all to replace/rewrite all outgoing in case it was not caught beforehand. Should therefor account for all cron emails, etc. going to _local_ domains.